main/dnd/dnd.py

```diff
@@ -47,13 +44,6 @@ def class_info(class_name):
     invoked_call = ctx.invoked_with.lower()
     if invoked_call in ctx.command.aliases:
         info = DND.class_info(invoked_call).json()
-    else:
```

Comment:
This actually could never trigger because of how `ctx.invoked_with` and `aliases` are connected in `discord.py`. I fixed it in the update.